File: archive/height-checker.js
const heightChecker = function(heights) {

  let memHeights = [...heights]
  heights.sort((a,b) => a-b)
  let ans = 0
  for (let i = 0; i < memHeights.length; i++) {
    if (memHeights[i]!==heights[i]) {ans++}
  }

  return ans
};

console.log(heightChecker([1,1,4,2,1,3]))

File: go-archive/height-checker.go
func heightChecker(heights []int) int {
    ans:=0

    nosorted:=append([]int{}, heights...)
    sort.SliceStable(heights, func (i int, j int)bool {
        return heights[i] < heights[j]
    })

    for i:=0; i < len(heights); i++ {
        if (heights[i] != nosorted[i]) {ans++}
    }
    return ans
}